cli-tests/README.md

```diff
@@ -0,0 +1,39 @@
+# PMM CLI tests
+Percona Monitoring and Management CLI automated tests. 
+
+
+## Getting Started
+
+* Open the _**cli**_ folder in console
+* Install Node.js 12+ version and make sure npx is included
+* Install project dependencies: `npm ci`
+* Install "playwright": `npx playwright install`
+* Install test module if required: `npm install -D @playwright/test`
+
+this is it! tests are good to go on specified PMM Server and/or Client.
+
+### Getting PMM server and client ready
+  * **Setup PMM Client**
+    * Download and unzip client tarball
+    * Put binaries into `/usr/bin` directory and grant corresponding permissions
+  * **Setup PMM Server**
+
+      _coming soon_
+      
+    * **Run CLI tests upon remote PMM server:**  
+      _coming soon_
```

Comment:
as far as I see it doesn't work against remote PMM Server yet, so let's remove this line to not have confusion.